### DISCUSSION TITLE
Production readiness of minifier?

### DISCUSSION THREAD

**Discussion by joebeachjoebeach** [swc-project/swc#6230] [Q&A] [answered]
Hello! Firstly I just want to say my team and I have been really happy with the performance gains we stand to make by switching from terser to swc. Our cold-cache JS builds will be 3 minutes faster (huge codebase), and average builds will be about 60s faster. We're excited about this, so I just wanted to say thanks! On to my question:

I was reading 강동윤's latest blog post about the tsc port, and in the [Moving Forward section at the end](https://kdy1.dev/posts/2022/10/tsc-port-status#moving-forward) I see "my focus continues to be swc, in particular making its minifier production-ready". Can you comment on its production readiness? Is there somewhere I can follow along with progress/milestones being made in that direction? I saw [this discussion](https://github.com/swc-project/swc/discussions/2933), which led me to [this issue](https://github.com/swc-project/swc/issues/2934), but neither of them seem to be super up to date.

**Comment by kdy1** [comment, marked as answer]
I think it's production ready already
